index.html

```diff
@@ -3,9 +3,84 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Josh Glantz Codes</title>
+    <link rel="stylesheet" href="style/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;700&display=swap" rel="stylesheet">
+
 </head>
 <body>
-    
+
+    <header>
+        <h2>JOSH GLANTZ</h2>
+        <nav>
+            <a href="index.html">Home</a><!--
+            --><a href="about.html">About</a><!--
+            --><a href="contact.html">Contact</a><!--
+            --><a href="projects.html">Projects</a>
+        </nav>
+    </header>
+    <section class="main">
+        <div class="img">
+            <img src="assets/andy-dutton-p9Hk-cbmIHo-unsplash.jpg" alt="Silhouette with blue sky">
+        </div>
+        <div class="blurb">
+            <h1>JOSH GLANTZ</h1>
```

Comment:
For accessibility, I like to leave the HTML as "normal" as possible and use CSS to stylize it. In this case, you can put `Josh Glantz` in the HTML, but then give it `text-transform: uppercase` to show it how you want. You could also experiment with `font-variant: small-caps` (instead of uppercasing. note that it is a different property though).

Reply:
Good to know. Both seemed rational at the time, but now that I know more, will do.